h5py_wrapper/wrapper.py

```diff
@@ -339,7 +339,9 @@ def _array_to_type(value, value_type):
                   'int': 'int',
                   'str': 'str',
                   'bool': 'bool',
-                  'Quantity': 'pq.Quantity'}
+                  'Quantity': 'pq.Quantity',
+                  'int64': 'np.int64',
+                  'float64': 'np.float64'}
```

Comment:
Good point. `np.int32` are detected as `np.int32` so using them would fail. The questions is whether we want to list all possible types explicitly, which would lead to a very long list ([nimpy docs](https://docs.scipy.org/doc/numpy/user/basics.types.html)). Or we just support `int64` and `float64` for now, because these are the ones that numpy picks automatically on a 64bit-system, which most systems nowadays are, I guess.